versioning/scripts/cloudbuild/main.go

```diff
@@ -54,6 +61,10 @@ const cloudBuildTemplateString = `steps:
       - 'build'
       - '--tag={{ .Tag }}'
       - '{{ .Directory }}'
+{{- if (eq $parallel true) }}
```

Comment:
should these just be `if $parallel`?

Reply:
Done.

I was trying to do something like that at beginning, but it failed. Now I cannot check difference.